apps/backend/api/src/routes/answer.route.ts

```diff
@@ -47,7 +47,7 @@ answerRouter.delete(
 
 answerRouter.get(
   '/',
-  validateSplit(undefined, undefined, Schema.AnswerWhereInputSchema),
```

Comment:
explain? has been tested in postman?

Reply:
I think yes I show you during the courses monday